cmd/helm/upgrade.go

```diff
@@ -127,6 +127,10 @@ func newUpgradeCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				}
 			}
 
+			if ch.Metadata.Deprecated {
+				fmt.Fprintln(out, "WARNING: Current chart is deprecated")
```

Comment:
Current chart is deprecated --> This chart is deprecated

Reply:
Fixed